projects/components/src/notification/notification.component.test.ts

```diff
@@ -1,10 +1,10 @@
 import { fakeAsync } from '@angular/core/testing';
 import { MAT_SNACK_BAR_DATA } from '@angular/material/snack-bar';
 import { IconType } from '@hypertrace/assets-library';
-import { IconComponent } from '@hypertrace/components';
```

Comment:
Does the new compiler catches these circular dependencies in test?

Reply:
Nope - I manually fixed all these because of the ngcc issue above. I thought the circular references breaking ngcc were our own, but it ended up being spectator. Since I had already fixed them, I left them in.